docs/docs/writing-hello-world.md

```diff
@@ -223,37 +185,36 @@ Assessed security level at 2020-07-17T16:31:51.894697Z is INSECURE
-change any time you alter the code of your enclave, the version of Conclave in use or the mode
-(simulation/debug/release) of the enclave. The enclave measurement should be stable across builds and machines, so
-clients can audit the enclave by repeating the Gradle build and comparing the value they get in the
-[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) against what the build process prints out.
-
-!!! tip
-    1. All this data is available via individual getters on the
-       [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) so you should
-       never feel a need to parse the output of `toString`.
-    2. `EnclaveInstanceInfo` is an interface so you can easily build mock attestations in your tests.
-    3. When not in simulation mode the timestamp is signed by Intel and comes from their servers.
-
-An instance has a [security assessment](api/-conclave%20-core/com.r3.conclave.common/-enclave-security-info/index.html), which can
-change in response to discovery of vulnerabilities in the
-infrastructure (i.e. without anything changing about the host or enclave itself). As we can see this enclave isn't
-actually considered secure yet because we're running in simulation mode still. An enclave can be `SECURE`, `STALE`,
-or `INSECURE`. An assessment of `STALE` means there is a software/firmware/microcode update available for the platform
-that improves security in some way. The client may wish to observe when this starts being reported and define a
-time span in which the remote enclave operator must upgrade.
-
-An attestation doesn't inherently expire but because the SGX ecosystem is always moving, client code will typically have
-some frequency with which it expects the host code to refresh the
-[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html). At present this is
-done by stopping/closing and then restarting the enclave.
+The hash in the first line is the *measurement*. This is a hash of the code of the enclave. It includes all
+the Java code inside the enclave as a fat-JAR, and all the support and JVM runtime code. This hash changes any time 
+you alter the code of your enclave, the version of Conclave in use, or the mode (simulation/debug/release) of the 
+enclave. The clients can audit the enclave by comparing the value they get in the
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) against what 
+the build process prints out.
+
+!!!Note
+
+    1. You can get all this data using individual getters on the
+       [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html). So 
+       you don't need to parse the output of `toString`.
+    2. You can easily build mock attestations in your tests using `EnclaveInstanceInfo` interface.
```

Comment:
Delete this bullet point. `EnclaveInstanceInfo` isn't designed to be extended by the user.